lib/reek/smells/smell_detector.rb

```diff
@@ -76,8 +74,8 @@ def default_smell_category
 
       attr_reader :smells_found # SMELL: only published for tests
```

Comment:
If we just use this in 1 tests, how about refactoring this test and get rid of it?